content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+6. After I obtained all the values that were required to build the request body to recover a cloud recovery point from `“0-Linux-Demo-VM02"`, I constructed the JSON request body as shown in the below figure. To restore the recovery point into a new virtual machine, the `restoreType` key of the request body JSON structure was set to `“ALTERNATE”` as shown in the below figure. I also provided the new virtual machine name after the recovery `“0-Linux-Demo-VN02-2-05-04-2024_05:48_PM”`. The [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/VirtualMachineRestore/) used for this:  
+
+   ```shellsession
+   POST /backup-recovery/v1beta1/virtual-machines/{{vmId}}/restore
+   ```
+
+![API to recover a cloud protection copy from a VM](/img/api-restoring-a-cloud-protection-recovery-point.png)
+
+7. To validate that the recovery was completed, and I tracked the progress from the response using the `async-operations` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/openapi/data-services-public-v1beta1/operation/GetAsyncOperation/) as shown below. The API used for this: 
+
+   ```shellsession
+   GET /data-services/v1beta1/async-operations/:id?select=associatedResources,createdAt,endedAt,error,displayName,healthStatus,id,customerId,progressPercent,name,type,state
+   ```
+
+![Task Id confirming the completion of the recovery](/img/task-display-recovery-is-completed.png)
+
+8. At the end, I went to the vCenter console to validate that the virtual machine 0-Linux-Demo-VN02-2-05-04-2024_05:48_PM was indeed part of the virtual machines inventory.
+
+![vCenter display the VM recovery completed](/img/the-vcenter-display-the-recovered-vm.png)
+
+## Summary
+
+This blog provides the introduction of the new set of REST API from the family of the APIs for data services on HPE Greenlake, namely **HPE GreenLake API for Backup and Recovery**. This set of API is documented at the HPE GreenLake for developer [website](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/) using interactive documentation based on OpenAPI version 3.1. Early in this blog post, I laid down the relationship of the resources in this HPE GreenLake API with the objects in the HPE GreenLake Backup and Recovery user interface. In this blog, I also introduced examples from several use cases associated with utilizing HPE GreenLake for Backup and Recovery to provide virtual machine protection from day one. The examples presented in this blog post provided some guides on using combination of the REST APIs that were announced in March 2024 to achieve the goal for protecting a virtual machine. All the execution for the examples were done using Postman API tool without any scripting language to encourage anyone to experiment with the family of REST API for data services on HPE GreenLake. 
```

Comment:
Modify to "This blog post introduces the new set of REST APIs in relation to the family of APIs for data services...

Second sentence, modify to "This set of APIs

Change "In this blog, I also introduced" to "In this post, I also introduced"